backend/alembic/versions/731e18130e01_questions_quiz_relationship.py

```
@@ -0,0 +1,50 @@
+    sa.ForeignKeyConstraint(['quiz_id'], ['quizzes.id'], ),
+    sa.PrimaryKeyConstraint('id')
+    )
+    op.drop_table('question')
+    # ### end Alembic commands ###
+
+
+def downgrade() -> None:
+    """Downgrade schema."""
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.create_table('question',
+    sa.Column('id', sa.INTEGER(), autoincrement=True, nullable=False),
```

Flake8 (7.2.0)

[error] continuation line under-indented for visual indent

(E128)